MetaMask/metamask-extension
# Part of #18714 and  #17670: Replacing deprecated constants

## Explanation

- This PR partially fixes following issues
- #18714 
- #17670 

Made changes to follow files

1. `ui/components/app/snaps/copyable/copyable.js`
2. `ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js`
3. `ui/components/app/snaps/snap-content-footer/snap-content-footer.js`
4. `ui/components/app/snaps/snap-version/snap-version.js`



## Screenshots/Screencaps





#### Created story for `Copyable` 
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/04dc1856-bb8f-444a-9995-f1feeaee37d1)
 On clicking the icon
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/2f12e682-4c82-4078-b390-05197dbabe98)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/1068b58f-a3b6-4b0c-93e5-d497efd8affc)

#### Created story for `SnapAuthorshipExpanded`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/085c6917-c71d-4528-8189-90e571acb568)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/49e86432-1181-4f1b-a11d-163bcfe00e8c)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/3663eb0a-64a7-488c-aa38-c93ae77b4e60)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/33dba717-da79-47f3-af14-0631dd584c69)


### Before
`snap-content-footer`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/d929d895-9ced-40a5-b14f-cbb3a8172f08)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/7bead89c-50d4-467e-8c32-18a81b990259)

`snap-version`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/d7cc942f-820c-4e7a-a18d-63ac12af746b)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/66abc12f-69b3-41d5-8c51-bee3304d41b3)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/1c977ceb-2329-4e27-9c2e-9ec1f9bde1d1)

### After


`snap-content-footer`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/ad83eb15-aade-4cc7-bdf3-652c3413bc2b)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/b057bda3-801b-495b-8966-ec52d621a014)

`snap-version`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/3ca3a0e5-7076-47ab-a650-d1f375bad024)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/b12f5072-5386-49f0-9cbd-b7c1e24e17d0)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/031c5b70-4440-4b35-8d3e-008696c4107d)

Covered `jest` test for `snap-version`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/dd996882-4ee9-4e11-b159-9cdcb80abe8d)

## Manual Testing Steps

1. Pull this branch
2. Run `yarn storybook`
3. Check following stories: `SnapContentFooter`, `SnapVersion`, `SnapAuthorshipExpanded`, `Copyable`



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
